Lib/test/datetimetester.py

```diff
@@ -2352,6 +2373,12 @@ def test_more_strftime(self):
             t = t.replace(tzinfo=tz)
             self.assertEqual(t.strftime("%z"), "-0200" + z)
 
+        # bpo-34482: Check that surrogates don't cause a crash.
```

Comment:
I don't think this belongs in the `test_more_strftime()` test method.

Reply:
Why? `TestDateTime.test_more_strftime` is a `datetime`-specific extension to `TestDate.strftime` (which in inherited by `TestDateTime`), and the latter contains my test for surrogates. I suggest  to either move all surrogate-related `strftime` tests to separate methods or keep things as is.

Reply:
I personally think breaking the tests up a bit more is a good idea, though I don't know the overall test philosophy of CPython.

If the idea is to put all the strftime tests in one test, I'd probably use subTest to distinguish the various failure modes, at the very least.